### PR TITLE
Asset page #49

### PR DESCRIPTION
**Pull request by yuliiagorgozka**

### CODE REVIEW

File: src/common/components/HeaderPage/HeaderPage.module.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  
+  .img {
+    width: 40px;
+    height: 40px;
+
+    margin-right: 8px;
+  }
+  .text {
+    font-size: 48px;
+    font-weight: 800;
```
Comment (DmitriyIvchenko) [inline src/common/components/HeaderPage/HeaderPage.module.scss:37]: variable exists for font-weight 800?

File: src/common/components/HeaderPage/HeaderPage.module.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+.link {
+  color: rgba(39, 198, 159, 1);
+  font-size: 16px;
+}
+
+.title {
+  display: flex;
+  align-items: center;
```
Comment (DmitriyIvchenko) [inline src/common/components/HeaderPage/HeaderPage.module.scss:26]: use mixing here

File: src/common/localization/translations/en.ts [hunk 3]
```diff
@@ -102,4 +102,7 @@ export default {
     title: 'Dashboard',
     secondaryTitle: 'Net Worth',
   },
+  Button: {
+    backDashboard: 'Back to Dashboard',
+  },
```
Comment (DmitriyIvchenko) [inline src/common/localization/translations/en.ts:107]: why key just button? only one button in system?

File: src/pages/HeaderCurrencyPage/HeaderCurrencyPage.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import iconsObj from 'assets/icons/iconsObj';
+import RoutePath from 'common/modules/routing/routing.enums';
+import {findById} from 'utils/helpers';
+
+function HeaderCurrencyPage() {
+  const theme = useTheme();
+  const {id} = useParams();
+  const translation = useTranslation();
+  const navigate = useNavigate();
+  const currency = findById(menuItems, id);
```
Comment (DmitriyIvchenko) [inline src/pages/HeaderCurrencyPage/HeaderCurrencyPage.tsx:18]: lodash -> _find(menuItems, {id})

**Review by DmitriyIvchenko — CHANGES_REQUESTED** [review]

File: src/common/components/HeaderPage/HeaderPage.module.scss (hunk 1 above)
Comment (yuliiagorgozka) [reply src/common/components/HeaderPage/HeaderPage.module.scss]: no

**Pull request merged**